### ISSUE TITLE
[Feature Request] Add more options to File Size display

### ISSUE THREAD

**Issue by Perkolator** [linuxmint/nemo#2505]
```
 * Nemo version (nemo --version): 4.6.5
 * Is issue with desktop or windowed nemo?: Don't understand the question.
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...): Mint 20.0
 * Graphics hardware *and* driver used: Intel 630 & NVIDIA RTX 2060 (mesa(?) & nvidia-driver-440(.100))
 * 32 or 64 bit: 64 bit
 ```

**Issue**
The file size display options are lacking. For example one might want to see all file sizes in the same prefix. The "(long format)" option isn't perfect for this, it shows too much information IMO, i.e. two prefixes, and thus isn't visually easy to see the size of the files, nor comparing file sizes of multiple files. I fear to write this, but Windows has it much better, the "KB" prefix for all files is just perfect, one can visually quickly see the sizes of the files, and the differences of the sizes of multiple files in the same directory. The KB precision (1 KB shown for all files that are smaller than that) isn't a hindrance IMO, personally I never have had to know the exact bytes size for a file, YMMV though. Other than KB/KiB prefix could be added too, give the user an option.

TL;DR:

- Option to display all file sizes in the same prefix, regardless how big the files might be.
- Add more prefix options, e.g. KB/KiB, MB/MiB.

Thank you!

**Comment by lixaotec** [comment]
God ! that is it!

Please, let do it, we need to improve alternatives such Nemo and Nautilus against Windows.

+1000000 KB thumbs to this feature

**Comment by lixaotec** [comment]
Also , colors of secondary columns (other than file name) should have text in grey color. as in Windows Explorer

And also date time format should not include Seconds, as in Windows Explorer.

I know these suggestions may appear lack of inovative, however we wish to keep with already is steady and productive. 

In a couple of years a OS will be just a File Manager and a Browser, nothing more, so we have to improve Linux on what matters already.

**Comment by ItzSwirlz** [comment]
I am sorry but this is not a ranting/demanding/hyping up forum. This is a place to make issues or suggest new features. Please be a bit more formal here for our developers, it’s really a lot more harder than you think.

**Comment by Perkolator** [comment]
> Also , colors of secondary columns (other than file name) should have text in grey color. as in Windows Explorer

I second this, it would be nice to have this.

> And also date time format should not include Seconds, as in Windows Explorer.

I second this too. If I remember right, Thunar allows to set a custom date format!

***

Both should be added as new feature requests?

**Comment by Jeremy7701** [comment]
The `ls` command is available with many colourful options for those who desperately need them.
No one would notice if nemo 'gained' these options.

**Comment by Pa9an** [comment]
Rather than only prefix options, I'd look for a simple "bytes" view. Means, sizes displayed like "1.234.567.890" instead of 1.2GB.

Comparing exact file sizes with Nemo is painful - the abovementioned view is missing and the long format is way too long (why duplicate kB/MB/GB **and** exacty size?). 

Also, mouseover on folder could calculate their size and display instead of the "25 items" in the size column.

**Comment by Perkolator** [comment]
> Also, mouseover on folder could calculate their size and display instead of the "25 items" in the size column.

There's lot of good new feature ideas in this topic. Should they all be added as new issues?

**Comment by Pa9an** [comment]
> > Also, mouseover on folder could calculate their size and display instead of the "25 items" in the size column.
> 
> There's lot of good new feature ideas in this topic. Should they all be added as new issues?
Sure, adding them as new (feature request) "issues" would be fine.

...coming from Directory Opus on both Amiga and Windows makes me pretty fastidious when it comes to file manager usability... ;-)

**Comment by Pa9an** [comment]
None of the suggestions (more prefixes, pure byte filesize, mouseover) have been implemented as of LM 22.3 Cinnamon.
Please help to identify way to get this through to somebody to listen and potentially consider.

**Comment by JosephMcc** [comment]
We no longer take feature requests here. This is where they should now be: https://github.com/orgs/linuxmint/discussions

After 6 years it's doubtful this will happen.

**Comment by Pa9an** [comment]
> We no longer take feature requests here. This is where they should now be: https://github.com/orgs/linuxmint/discussions
> 
> After 6 years it's doubtful this will happen.

...which is a shame, given the simplicity of this request.

Will request it again under https://github.com/orgs/linuxmint/discussions, thanks for the info.
